ConfigurationSystem/Client/Utilities.py

```diff
@@ -463,6 +465,71 @@ def getGridSRMs(vo, bdiiInfo=None, srmBlackList=None, unUsed=False):
   return S_OK(srmSeDict)
 
 
+def getBDIISEs(vo, bdiiInfo=None, sesBlackList=None, unUsed=False):
+  """ Get all the SRMs available for a given VO
+
+      :param str vo: VO name
+      :param dict bdiiInfo: information from BDII
+      :param list sesBlackList: SRMs from black list
+      :param bool unUsed: unused
+
+      :return: S_OK(dict)/S_ERROR()
+  """
+  result = ldapService(serviceType='???', vo=vo)
```

Comment:
@andresailer @atsareg what should I use here for getting all SEs types?

Reply:
For GLUE2 I would guess it is not that easy.

Reply:
Here for reference the GridPP code: https://github.com/ic-hep/GridPPDIRAC/blob/master/ConfigurationSystem/private/AutoBDIISEs.py

They seem to look also for `xroot` with:
```
    xrootports = ldap_conn.search_s(base=base, scope=scope,
                                    filterstr="(&(objectClass=GlueSEAccessProtocol)"
                                    "(GlueChunkKey=*)"
                                    "(GlueSEAccessProtocolEndpoint=*)"
                                    "(|(GlueSEAccessProtocolType=Root)"
                                    "(GlueSEAccessProtocolType=Xroot)))")
```